src/CMakeLists.txt

```diff
@@ -39,10 +39,12 @@ if( POLICY CMP0054 )
   cmake_policy(SET CMP0054 OLD)
 endif()
 
+get_filename_component(REPOSITORY_DIR ${PROJECT_SOURCE_DIR}/.. ABSOLUTE)
+
 #
 # Include extra CMake files
 #
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/cmake)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${REPOSITORY_DIR}/external)
```

Comment:
This line should be removed. This line should be changed to include `PARENT_SCOPE` at the end instead:

https://github.com/numenta/nupic.core/blob/e3dfdb262fbba3158859b561c8d39f4e87d570d1/external/CMakeLists.txt#L46